src/js/edu-benefits/containers/SubmitPage.jsx

```diff
@@ -7,28 +7,22 @@ import { relinquishableBenefits } from '../utils/options-for-select';
 
 class SubmitPage extends React.Component {
   render() {
-    const { relinquished, benefits, confirmation, address, timestamp } = this.props;
-
-		// TODO: replace this with actual data from the API
     return (
-      <SubmitMessage
-          claimType="Education Benefits"
-          confirmation={confirmation}
-          date={timestamp}
-          address={address}
-          claimedBenefits={benefits}
-          relinquishedBenefits={relinquished}/>
+      <SubmitMessage claimType="Education Benefits" {...this.props}/>
     );
   }
 }
 
 function mapStateToProps(state) {
+  // TODO: replace this with actual data from the API
```

Comment:
It seems a little weird to me that we populate it with data from the client.  I suppose it's fine-- the UX doesn't change either way.

Comment removed.